lib/browser/rpc-server.js

```diff
@@ -449,3 +450,23 @@ ipcMain.on('ELECTRON_BROWSER_WINDOW_CLOSE', function (event) {
   }
   event.returnValue = null
 })
+
+ipcMain.on('ELECTRON_BROWSER_SANDBOX_LOAD', function (event, preloadPath) {
+  let preloadSrc = null
+  let preloadError = null
+  if (preloadPath) {
+    try {
+      preloadSrc = fs.readFileSync(preloadPath).toString()
+    } catch (err) {
+      preloadError = {stack: err.stack}
```

Comment:
I hate to be that person 😆 But welcome to javascript where you can `throw null`

This should check if we can get to `.stack` before actually trying or our error handling will error 😆

Reply:
:+1: